documentation/library_guide/conf.py

```diff
@@ -112,12 +103,12 @@
 # html_theme = 'otc_tcs_sphinx_theme'
 # html_theme_path = ['_themes']
 
-import sys
-import os
+# import sys
```

Comment:
In the #156 PR, I removed some of the commented out changes. There will be further cleanup for 2021.3. I had not removed them just in case, but agree that a cleaner copy is better.